packages/experimental/scripts/generate/templates/DISPLAY_NAME.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import classnames from 'classnames';
```

Comment:
Personally prefer the use of 'cx' but happy to use 'classNames' as per the example here https://www.npmjs.com/package/classnames.

We should update other experimental components as appropriate, perhaps after a brief slack or sync discussion.

Reply:
My care-amount is low on this - I went with Carbon's lead